books/free-programming-books-langs.md

```diff
@@ -2170,6 +2170,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [SQL For Web Nerds](http://philip.greenspun.com/sql/)
 * [SQL Notes for Professionals](http://goalkicker.com/SQLBook/) - Compiled from StackOverflow Documentation (PDF)
 * [SQL Queries Succinctly](https://www.syncfusion.com/ebooks/sql_queries_succinctly) - Nick Harrison
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite)
+* ```
```

Comment:
```suggestion
```

Reply:
I use the suggestions for quick editing of these long files and I think I pressed the suggestion button twice without cleaning/removing the extra backticks!